front/src/applications/stdcm/views/OSRDStdcmConfig.tsx

```diff
@@ -155,13 +159,15 @@ export default function OSRDConfig({
                         : 'stdcm-map-noSimulation'
                     }`}
                   >
-                    <Map />
+                    <Map setMapCanvas={setMapCanvas} />
                   </div>
                 )}
               </div>
             </div>
             {currentStdcmRequestStatus === STDCM_REQUEST_STATUS.success &&
-              shouldDisplayStdcmResult && <OSRDStdcmResults />}
+              shouldDisplayStdcmResult && (
```

Comment:
Make stdcmResults not optional in OSRDStdcmResults. Remove shouldDisplayStdcmResult. Also give rollingStock as a param in OSRDStdcmResults
```suggestion
              stdcmResults && rollingStock && (
```

Reply:
done